### ISSUE TITLE
Problem with looping the http://zxing.appspot.com/scan function

### ISSUE THREAD

**Issue by cooperk937** [zxing/zxing#844]
Hi,

I love ZXing and have used it for a few projects now. My most recent problem is a bit trickier. I want to make a web browser that links to http://zxing.appspot.com/scan to retrieve barcode data.

It works perfectly, until I try to run the function twice, even with a few seconds gap in between.

To prove my error, I have created 2 files; index.html and return.html and you can experience this here:
http://karl-cooper.co.uk/zxingtest/v1/

The first one always works, the second one never does, you are greeted with 'you need to install free Barcode Scanner -app' even though I obviously have it as I used it seconds ago.

File sources are as follows:
**index.html (this is used to start the loop)**
`<a style="font-size:52px;" href="http://zxing.appspot.com/scan?ret=http%3A%2F%2Fwww.karl-cooper.co.uk%2Fzxingtest%2Fv1%2Freturn.html%3Fcode%3D%7BCODE%7D&SCAN_FORMATS=CODE_39,CODE_93,CODE_128,DATA_MATRIX,ITF,CODABAR,EAN_13,EAN_8,UPC_A,QR_CODE">Click here to start recursive scanning</a>`

**return.html (this is used to continue the loop)**
`Thank you for the data from the barcode, now I will send the user back to read another!
<script>
	setInterval(function() { window.location.href = "http://zxing.appspot.com/scan?ret=http%3A%2F%2Fwww.karl-cooper.co.uk%2Fzxingtest%2Fv1%2Freturn.html%3Fcode%3D%7BCODE%7D&SCAN_FORMATS=CODE_39,CODE_93,CODE_128,DATA_MATRIX,ITF,CODABAR,EAN_13,EAN_8,UPC_A,QR_CODE"; }, 3000 );
</script>`

PLEASE NOTE THAT I TESTED THIS ON A SAMSUNG GALAXY S7 AND IT ISN'T WORKING.

**Comment by srowen** [comment]
I don't think that will work, but it's not related to the app. URIs you open that way aren't handled by Intents, I think. But if you can suggest a concrete change, reopen this.
